### PR TITLE
fix: 🐛 Fixed typescript build

### PR DESCRIPTION
**Pull request by kostysh**
This PR:

- fixes not worked typescript `build`
- updates some dependencies versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "lint": "npx eslint . --ext .ts",
     "lint:fix": "npx eslint . --ext .ts --fix",
     "test": "npx mocha -r ts-node/register test/**/*.spec.ts",
-    "build": "npx tsc",
+    "build": "rm -rf dist && npx tsc -p tsconfig-build.json",
```
Comment (mfw78) [inline package.json:31]: Perhaps we could create a `clean` script as well, such that:

```json
"clean": "rm -rf dist",
"build": "npx clean && npx tsc -p tsconfig-build.json"
```

This way we can have a logical place to insert these conditions?

File: tsconfig.json [hunk 2]
```diff
@@ -21,6 +21,9 @@
     "./test",
     "./swagger"
   ],
+  "exclude": [
+    "node_modules"
```
Comment (mfw78) [inline tsconfig.json:25]: Is this needed for some specific reason? Given that this is the default per https://www.typescriptlang.org/tsconfig#exclude

**Review by mfw78 — CHANGES_REQUESTED** [review]

**Comment by mfw78** [comment]
@kostysh With respect to `CI/CD` testing, if these are related to environment variables, how about we use something like:

```json
"test": "source .env.example && yarn test "
```

Or something like this. This way we can both:

1. Be ensured that the `.env.example` that we are packaging is valid.
2. Our test environment doesn't depend on GitHub CI/CD environment.

File: package.json (hunk 1 above)
Comment (kostysh) [reply package.json]: - [x] done

File: tsconfig.json (hunk 2 above)
Comment (kostysh) [reply tsconfig.json]: - [x] done

**Review by mfw78 — APPROVED** [review]

**Pull request merged**